### DISCUSSION TITLE
How to view logs on Raspbian Lite Slinger?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#364] [Q&A] [answered]
What's the command for running and viewing logs for Slinger on a Raspbian Lite over SSH?

**Comment by regmar505** [comment, marked as answer]
ssh pi@<pi_address>
more /tmp/sling.log

**Reply by GR0DZ1LLA** [reply]
you can also `tail -f` the file to view it in real time
